metomi/rosie/ws.py

```diff
@@ -204,7 +211,7 @@ def get(self, *args):
         except (KeyError, AttributeError, jinja2.exceptions.TemplateError):
             import traceback
             traceback.print_exc()
-        except rosie.db.RosieDatabaseConnectError as exc:
+        except metomi.rosie.db.RosieDatabaseConnectError as exc:
             raise tornado.web.HTTPError(404, str(exc))
 
     def _render(self, all_revs=0, data=None, filters=None, s=None):
```

Comment:
Do we need `wsgi` module somewhere in `setup.py` as optional dependency too? (Used in the main method here I think)

Reply:
I think that we don't need a wsgi module, and that a wsg interface is provided by the tornado module. Perhaps @sadielbartholomew  should comment - I think it's her code.